package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "electron .",
     "buildwindows": "electron-builder build --win --x64 --ia32 --publish never",
-    "buildlinux": "electron-builder build --linux deb tar.gz --publish never",
+    "buildlinux": "electron-builder build --linux deb pacman tar.gz --publish never",
```

Comment:
i have no clue here, but it might be better to split the pacman build into his own command/script "buildpacman" or something like that
that way rob or whoever builds the releases can decide to do one or the other, or both ;)
and existing automation/scirpts/etc. that use the "buildlinux" command/script will still get their expected result